src/util/time.js

```diff
@@ -175,33 +180,32 @@ function isValidCronPattern (pattern) {
  * @param {String} [attrs.timezone='UTC'] - https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
  * @returns {String[]} ISO Dates
  */
-function computeRecurringDates (pattern, { startDate, endDate, timezone = 'UTC' } = {}) {
+function computeRecurringDates (pattern, { startDate, endDate, timezone } = {}) {
   if (!isDateString(startDate) || !isDateString(endDate)) {
     throw new Error('Expected start and end dates')
   }
   if (endDate < startDate) {
     throw new Error('Invalid dates')
   }
 
-  const cronInstance = new CronConverter({
-    timezone
-  })
-  cronInstance.fromString(pattern)
+  timezone = timezone || 'UTC'
 
-  const schedule = cronInstance.schedule(startDate)
+  const cronTime = new CronTime(pattern, timezone)
 
   let continueLoop = true
   const dates = []
+  let cronISODate = new Date(new Date(startDate).getTime() - 1) // start from `startDate` minus 1 millisecond
 
   while (continueLoop) {
-    const momentDate = schedule.next()
-
-    const date = momentDate.toISOString()
+    // `_getNextDateFrom` is a private method from `CronTime.prototype`
+    // Please check its availability when upgrading the library `cron`
+    const cronMomentDate = cronTime._getNextDateFrom(cronISODate, timezone)
```

Comment:
I don’t see this function in your tests. This would help to detect problems with updates.

Reply:
Indeed, I'll add a test for it.